### PR TITLE
Add Brett McLaren as a committer.

### PR DESCRIPTION
**Pull request by johnkord**
Brett just recently joined my team and I'm looking for him to pick up some issues related to packaging/build/CI tasks.

This is a request to the CMC (Community Maintenance Committee) to add Brett as a committer to the Open Enclave project. I've added each maintainer to the reviewers for this PR.

### CODE REVIEW

File: docs/Maintainers.md [hunk 1]
```diff
@@ -93,3 +93,4 @@ instead, it should be brought up with the Community Maintenance Committee.
 | Shruti Ratnam         | shruti25ratnam      | Attestation                    |
 | Cheng-mean Liu        | soccerGB            | Attestation, SGX               |
 | Bruce Campbell        | yakman2020          | Windows, SGX                   |
+| Brett McLaren         | BRMcLaren           |                                |
```
Comment (dthaler) [inline docs/Maintainers.md]: Should the areas say packaging/build/CI instead of being blank?

Comment (johnkord) [reply docs/Maintainers.md]: @dthaler Brett may end up working on some other tasks unrelated to packaging/build/CI, so I left it blank.
I'm genuinely not really sure what to put here for new people! What does everyone think? :)

Comment (dthaler) [reply docs/Maintainers.md]: If it's blank, it means the person doesn't need to be added as a reviewer on anything.
If it's not blank, it means the person should be added as a review on anything in that area.

Comment (johnkord) [reply docs/Maintainers.md]: Sounds good to me. For now I'd say lets leave it blank, since Brett is just now onboarding.  Thanks so much, Dave!

**Comment by johnkord** [comment]
Thanks for your quick approval, everyone.

bors r+

**Comment by andschwa** [comment]
Hm, we should probably leave something like this open for more than a few hours. I’ll add to the agenda a consideration of required holding time for decisions (this being something that at least a few of us didn’t have a chance to see today).

**Pull request merged**